libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,152 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.3>>
 * <<breaking-changes-6.0>>
 * {auditbeat}/auditbeat-breaking-changes.html[Breaking changes in Auditbeat 6.2]
 
+[[breaking-changes-6.3]]
+=== Breaking changes in 6.3
+
+This section discusses the main changes that you should be aware of if you
+upgrade the Beats to version 6.3. Please also review the relevant
+Breaking Changes sections of the <<release-notes,release notes>>.
+
+[[breaking-changes-mapping-conflict]]
+==== New `host` namespace may cause mapping conflicts for Logstash
+
+// REVIEWERS: I describe these as mapping conflicts throughout, but the acutal
+// error message says mapper parsing exception. I'm wondering if I should use
+// that instead?
+
+This breaking change applies only to users who send Beats events to Logstash.
+
+Starting with version 6.3, Beats provides an `add_host_metadata` processor for
+adding fields, such as `host.name` and `host.id`, to Beats events. These fields
+are grouped under a `host` prefix and conform to the
+https://github.com/elastic/ecs[Elastic Common Schema (ECS)]. The `host` object
+is defined in the Elasticsearch index template even if the processor is not
+used.
+
+We've also added a `host.name` field to all events sent by Beats. This field
+prevents the Beats input plugin in Logstash from adding a default `host` field.
+(By default, the plugin adds a `host` field if the event doesn't already have
+one. We don't want the plugin to add this field because it causes a mapping
+conflict with the `host` object defined in the index template.)
+
```

Comment:
The first sentence in the parentheses doesn't say anything new, so consider deleting it, and adding `host` to remaining sentence.

+(We don't want the plugin to add the `host` field because it causes a mapping
+conflict with the `host` object defined in the index template.)

Reply:
I had it that way originally, but thought being explicit might be useful for users who don't understand that LS adds the field if there's not one. This behavior isn't consistent across input plugins, so it might be worth calling out. But I agree...it's a bit redundant--that's why I made it parenthetical even though I know people hate parens. If no one disagrees, I will remove the statement.